code/signup.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Project 3</title>
+
+  <!-- Google font import -->
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+
+  <!-- CSS Import -->
+  <link rel="stylesheet" href="./style.css">
+
+</head>
+
+<body>
+
+  <!-- Header container start -->
+  <div class="headerContainer">
+
+    <div class="navigation">
+
+      <div class="TabletDesktopMenu">
+        <a class="menuLink" href="./index.html">Home</a>
+        <a class="menuLink" href="./signup.html">Sign up</a>
+      </div>
+      <!-- Mobile Hamburger menu start -->
+      <div>
+        <button class="ham"></button>
+        <nav class="navbar">
+          <ul>
+            <li><a class="menuLink" href="./index.html">Home</a></li>
+            <li><a class="menuLink" href="./signup.html">Sign up</a></li>
+          </ul>
+        </nav>
+      </div>
+      <!-- Mobile hamburger manu end -->
+
+      <!-- Header title start -->
+      <div>
+        <h2>What the stock</h2>
+      </div>
+      <!-- Header title end -->
+
+    </div>
+
+    <!-- Hero image and text start -->
+    <div class="heroContainer">
+      <!-- Hero Image -->
+      <img src="./images/heroImage.jpg" alt="" class="headerImage">
+
+      <div class="heroTitle">
+        <!-- Header Title -->
+        <h1>Use the form below to sign up</h1>
+
+      </div>
+    </div>
+    <!-- Hero image and text end -->
+  </div>
+  <!-- Header container end -->
+
+
+
+  <!-- Signup form start -->
+  <div class="signupFormFlexbox">
+    <div class="signupFormStyling">
+      <form action="https://httpbin.org/anything" type="POST">
+        <h1 style="text-align: center; padding-bottom: 10px;">Sign up</h1>
```

Comment:
Try to keep the styling to the CSS file

Reply:
I was thinking that for a one off styling i could set it directly in the HTML thus leaving the css free of unneccessary lines. Is it considered incorrect? I'll change it for now of course! 👍🏻

Reply:
The same argument could go for keeping the CSS to the CSS file (keeping the HTML CSS free and "cleaner") ;)